include/boost/http_proto/serializer.hpp

```diff
@@ -293,48 +338,147 @@ class BOOST_SYMBOL_VISIBLE
     bool is_done_;
     bool is_chunked_;
     bool is_expect_continue_;
+    bool is_compressed_ = false;
 };
 
 //------------------------------------------------
 
+/**
+    A proxy type used to pass bounded input to the
+    associated serializer.
+*/
 struct serializer::stream
 {
-    /** Constructor.
+    /** Default constructor.
```

Comment:
I prefer "Constructor" so that it groups with the other constructors in the document generation tools. Otherwise it will be in its own group.